package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "prisma-typebox-generator",
-  "version": "2.0.2",
+  "name": "jsbrain-prisma-typebox-generator",
```

Comment:
please dont change this unless you want to fork it and create different distribution